app/uk/gov/hmrc/customs/notification/repo/LockRepo.scala

```diff
@@ -16,22 +16,73 @@
 
 package uk.gov.hmrc.customs.notification.repo
 
-import org.joda.time.DateTime
-import uk.gov.hmrc.customs.notification.domain.{ClientNotification, ClientSubscriptionId}
 
-import scala.concurrent.Future
-import scala.concurrent.duration.Duration
+import org.joda.time.Duration
+import reactivemongo.api.DB
+import uk.gov.hmrc.customs.notification.domain.ClientSubscriptionId
+import uk.gov.hmrc.lock.{ExclusiveTimePeriodLock, LockRepository}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{ExecutionContext, Future}
+
```

Comment:
how about `val id: String` so that usage reads `lockOwner.id` rather than `lockOwner.lockOwnerId`

Reply:
yep will change